src/lib/runtime/Web3FunctionRunner.ts

```diff
@@ -267,18 +286,40 @@ export class Web3FunctionRunner {
           memory,
           rpcCalls,
           network: networkStats,
+          throttled,
         };
       }
     } else {
+      if (error && error instanceof Web3FunctionRuntimeError) {
+        switch (error.throttledReason) {
+          case "memory":
+            throttled.memory = error.message.includes("Memory limit exceeded");
+            break;
+          case "rpcRequest":
+            throttled.rpcRequest = error.message.includes(
+              "RPC requests limit exceeded"
+            );
+            break;
+          case "duration":
+            throttled.duration = error.message.includes(
+              "exceed execution timeout"
+            );
+            break;
+          default:
+            break;
+        }
```

Comment:
We should eliminate the `message.includes` as we don't need them anymore.
What about this suggestion?
```suggestion
        throttled[error.throttledReason] = true;
```

Reply:
fixed in 7abfd4c68c102a9fea79d114886fbbe6c5017823